amethyst_network/src/connection.rs

```diff
@@ -32,12 +34,13 @@ pub struct NetConnection<E: 'static> {
 
 impl<E: Send + Sync + 'static> NetConnection<E> {
     /// Construct a new NetConnection. `SocketAddr` is the address that will be connected to.
-    pub fn new(target: SocketAddr) -> Self {
+    pub fn new(target_receiver: SocketAddr, target_sender: SocketAddr) -> Self {
```

Comment:
I leave it since the connection created heer is one that simulates some connection to some endpoint. By having `target` it sounds clearer to me that you want to specify the target.